tt_datastore/datastore.py

```diff
@@ -25,70 +31,128 @@ def __init__(self, host, username, password, bucket, **kwargs):
 
         queryStr = "&".join(connectArgs)
         if queryStr:
-            connectionString += "/?{0}".format(queryStr)
+            connectionString += "?{0}".format(queryStr)
 
-        cluster = Cluster(connectionString)
         authenticator = PasswordAuthenticator(username, password)
-        cluster.authenticate(authenticator)
-        kwargs['quiet'] = True
-        self.bucket = cluster.open_bucket(bucket, **kwargs)
+        self.cluster = Cluster(connectionString, ClusterOptions(authenticator))
+        self.bucket = self.cluster.bucket(bucket)
+        self.viewManager = self.bucket.view_indexes()
+        self.bucketManager = BucketManager(self.bucket._admin)
+        self.queryManager = self.cluster.query_indexes()
+        self.collection = self.cluster.bucket(bucket).default_collection()
```

Comment:
Could we still apply quiet=True here?

Reply:
It's not well documented, but as far as I can tell, in the 2.5 -> 3.0 migration they took that functionality away at the cluster/bucket level, so you have to set it individually when invoking the relevant methods.